src/main/agent/agent.ts

```diff
@@ -869,11 +857,7 @@ export class Agent {
       }
 
       logger.error('Error running prompt:', error);
-      if (error instanceof Error && (error.message.includes('API key') || error.message.includes('credentials'))) {
-        project.addLogMessage('error', `${error.message}. Configure credentials in the Settings -> Providers.`, false, promptContext);
-      } else {
-        project.addLogMessage('error', `${error instanceof Error ? error.message : String(error)}`, false, promptContext);
-      }
+      project.addLogMessage('error', `${error instanceof Error ? error.message : String(error)}`, false, promptContext);
```

Comment:
```
if (error instanceof Error && (error.message.includes('API key') || error.message.includes('credentials'))) {
        project.addLogMessage('error', `${error.message}. Configure credentials in the Settings -> Providers.`, false, promptContext);
      }
```

The error should still be checked for these strings, I believe. This is to give a more descriptive message to user.